generators/server/templates/src/test/java/package/web/rest/UserResourceIT.java.ejs

```diff
@@ -552,8 +555,11 @@ public class UserResourceIT <% if (databaseType === 'cassandra') { %>extends Abs
             <%_ } _%>
             .jsonPath("$.langKey").isEqualTo(DEFAULT_LANGKEY);
         <%_ } _%>
-        <%_ if (cacheManagerIsAvailable === true) { _%>
 
+        <%_ if (cacheProvider === 'memcached') {  _%>
+        if(!(cacheManager instanceof NoOpCacheManager))
```

Comment:
format, need space, need { and }

Reply:
Now, that's a good eye :) How, did I miss the missing {} :hushed:

Reply:
Indeed I used my own style of coding there which isn't that popular.
I've tried fixing it but I am unsure if it fits the standart